### ISSUE TITLE
Alternate manga view

### ISSUE THREAD

**Issue by RillonDodgers** [Aidoku/Aidoku#254]
Would it be possible to have a view like the screenshot when browsing or in the library of seeing manga? My argument for this feature is that you would be able to put chapter counts, statuses, etc without having to open each manga to check them. I personally don't like starting a manga with little to no chapters just because I can get through 80+ chapters relatively quick, and it'd be easier to see new ones to start with a feature like this.

I don't know what kind of data you're currently getting on those requests so I understand if you didn't want to have to make more requests just to show this data in the view.

![IMG_5777](https://github.com/Aidoku/Aidoku/assets/35751318/bc6c23e4-e638-453a-a02c-9e0f3f2c8d46)

**Comment by Skittyblock** [comment]
This is something I originally wanted to do (and had implemented at one point), but the problem with it is that it does indeed take an extra call for each manga on top of the original call for just the list. If we added chapter count, it would then double that for each manga with how sources currently work.

**Comment by RillonDodgers** [comment]
That makes complete sense and figured that's why it hadn't been implemented!

If it's not able to have this view in the browse section of the app, is it possible to have it within the library section? I'm not sure how the user data is structured but I imagine the metadata is stored locally once it's added to the library.

I 100% respect any decision! Just throwing ideas out there that may or may not stick. Thanks!

**Comment by Skittyblock** [comment]
Yeah I removed the list view for the library when I switched from SwiftUI to UIKit and never got around to rewriting it. I'm planning on doing it eventually.
